### ISSUE TITLE
[Feature Request]: Notify users if they're using an older version

### ISSUE THREAD

**Issue by jamesgeorge007** [nuxt/create-nuxt-app#406]
Users may not be aware about any newer releases being made and would end up using the older versions.

[update-notifier](https://www.npmjs.com/package/update-notifier) serves the purpose here. As from the tagline, `Inform users of your package of updates in a non-intrusive way`.

**Comment by vinayakkulkarni** [comment]
if you use `npx` instead of globally installing `npm i -g create-nuxt-app` & use `create-nuxt-app <app>`, wouldn't `npx` already pull the latest version of cna and scaffold a project for you?

**Comment by jamesgeorge007** [comment]
Yeah, good point. But, consider this scenario if a user has installed `CNA` globally so that he can scaffold `Nuxt.js` projects anytime he wishes for without needing to install the package again. There is a chance that he/she might not be aware of a newer release. This is where `update-notifier` comes handy.

**Comment by jamesgeorge007** [comment]
\cc @clarkdo @Atinux

**Comment by clarkdo** [comment]
@jamesgeorge007 Thanks, I'll have a look at this request

**Comment by jamesgeorge007** [comment]
Can I work on this?

**Comment by clarkdo** [comment]
I think so, but let ask if @Atinux and @pi0 have any thought.

**Comment by adrianoresende** [comment]
With `npm i -g create-nuxt-app` & `create-nuxt-app client` or `npx create-nuxt-app client`
It was generated like this:

![image](https://user-images.githubusercontent.com/798755/78727944-6b995200-790c-11ea-8462-82723acc9a2f.png)

Several outdated versions, that's right?

**Comment by manniL** [comment]
Because your globally installed version of `create-nuxt-app` is outdated.

I recommend to use `npx` all the time and not installing it via `npm -g`. Otherwise that version will be used all the time and has to be kept up to date via `npm update -g`

**Comment by vinayakkulkarni** [comment]
The @vue/cli already has feature like this:
<img width="337" alt="Screenshot 2020-04-11 at 9 05 39 PM" src="https://user-images.githubusercontent.com/19776877/79048100-4a03d780-7c38-11ea-9051-fcd2df2ed6dd.png">
